opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew.adoc

```diff
@@ -1,205 +1,103 @@
-| `opennms-threshold-states:clear-all` | `opennms:threshold-clear-all` 
-| `opennms-threshold-states:clear` | `opennms:threshold-clear` 
-| `opennms-meta:test` | `opennms:metadata-test` 
-| `opennms-rpc:stress` | `opennms:stress-rpc` 
-| `opennms-kafka-sink:topics` | `opennms:kafka-sink-topics` 
-| `opennms-kafka-rpc:topics` | `opennms:kafka-rpc-topics` 
-| `opennms-snmp:fit` | `opennms:snmp-fit` 
-| `opennms-snmp:show-config` | `opennms:snmp-show-config` 
-| `opennms-snmp:remove-from-definition` | `opennms:snmp-remove-from-definition` 
-| `opennms-snmp:walk` | `opennms:snmp-walk` 
-| `snmp:local-engine-id` | `opennms:snmp-local-engine-id` 
-| `opennms-health:metrics-list` | `opennms:metrics-list` 
-| `opennms-health:metrics-display` | `opennms:metrics-display` 
-| `opennms-health:check` | `opennms:health-check` 
-| `opennms-graph:get` | `opennms:graph-get` 
-| `opennms-graph:list` | `opennms:graph-list` 
-| `opennms-graph:force-reload` | `opennms:graph-force-reload` 
-| `opennms-graph:search` | `opennms:graph-search` 
-|===
+* *PostgreSQL 10 or higher*: Horizon 27 requires any supported version of PostgreSQL from 10 up to (and including) 13.
+
+== Breaking Changes
+
+=== Remote Poller Removed
+
+The remote poller has been removed from Horizon 27.
+It has been replaced with new infrastructure for <<Application Perspective Monitoring,perspective-based monitoring>> that runs on Minions.
+
+=== VMware Assets
+
+VMware-related data is not longer stored in node assets.
+The `install` command will move any existing VMware asset data to the node's metadata when you upgrade.
 
 == New Features and Improvements
 
-=== Documentation Updates
+=== Karaf Shell
 
-Documentation improvements have been made in a number of areas:
+A number of useful commands have been added or updated in the Karaf shell.
 
-* Docker
-* Grafana PDF Reports
-* Graph Service API
-* Minion
-* Telemetry/flows
-* Thresholding
-* Timeseries Integration
+* `alarm-list`: list current alarms (optionally filtered by UEI)
+* `event-list`: list events (optionally filtered by UEI)
+* `ip-addr-range`: generate a list of IP addresses to use in other commands
+* `reload-daemon`: now accepts an optional configuration file to target for reload
+* `send-event`: additional options supported to match the CLI version of `send-event` more closely
+* `snmp-fit`: can now optionally send a `newSuspect` event on success
+* `snmp-get`: like `snmp-walk`, but fetches a single MIB object and prints the result
 
-=== Graph API
+=== Minion and Sentinel
 
-A new API has been added for querying graph data, inspired by link:http://graphml.graphdrawing.org[GraphML].  
-This API consolidates a number of internal APIs for accessing topology, application, BSM, link, and other data.
+* The Minion `confd` configuration support now handles a bunch more options, including setting Jetty, RMI, and ssh host and port, Secure Credentials Vault configuration, and environment variables and options for the JVM.
```

Comment:
If I recall correctly we can reference to the [CONFD_README](https://github.com/OpenNMS/opennms/blob/develop/opennms-container/minion/CONFD_README.md) which is the only place which documents what is possible with Minion and Confd.

Reply:
Good point, I'll link it in the release notes.  We should probably make an issue to transition that to actual asciidoc.